packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -62,6 +62,8 @@ export function defaultSerialize(
       object[name] = get(model, name);
     } else if (/_id$/.test(name)) {
       object[name] = get(model, 'id');
+    } else {
+      object[name] = get(model, name);
```

Comment:
Conceptually this is pretty odd (e.g. if we were going to fall back to use `get(model, name)` then we shouldn't check `name in model` at the first branch above)

I think we could try to o something special like:

```js
} else if (isProxy(model)) {
  object[name] = get(model, name);
}
```

Reply:
@rwjblue as long as the helper util doesn't check for a symbol-ish thing that EmberData can't easily make use of (e.g. it checks for the well known proxy methods) then I don't see an issue with doing it this way.

Reply:
@runspired ya, its basically anything that uses the proxy mixin

https://github.com/emberjs/ember.js/blob/23ee5d1a8aa63678f680392e2d5f3d035873e75c/packages/%40ember/-internals/runtime/lib/mixins/-proxy.js#L76